charts/jaeger/templates/collector-ing.yaml

```diff
@@ -14,14 +13,14 @@ metadata:
   {{- end }}
 spec:
   rules:
-    {{- range $host := .Values.collector.ingress.hosts }}
-    - host: {{ $host }}
+    {{- range .Values.collector.ingress.hosts }}
+    - host: {{ .host }}
       http:
         paths:
           - path: {{ $basePath }}
             backend:
               serviceName: {{ template "jaeger.collector.name" $ }}
-              servicePort: {{ $servicePort }}
+              servicePort: {{ .servicePort }}
```

Comment:
Would it make sense to fallback to `.Values.collector.service.http.port` which is the current value for folks who are not explicitly setting the servicePort?

Reply:
That would be an option. However, personally I think it should be a conscious decision which port to use. Since the value for `host` must be configured anyway and the `servicePort` value is right next to it (with an example given) I think there is not that much benefit for a fallback.

Reply:
I would agree if the ingress collector was not already implemented, but my concern is to not introduce breakage to users who are using the collector ingress today.

Reply:
Ok, this makes sense. I will take a look at it tonight.

Reply:
I changed it so that `.Values.collector.service.http.port` is now the fallback.